cli/CHANGELOG.md

```diff
@@ -24,6 +25,7 @@ _Released 02/10/2023_
 
 - Fixed a regression introduced in Cypress [12.5.0](https://docs.cypress.io/guides/references/changelog#12-5-0) where the `runnable` was not included in the [`test:after:run`](https://docs.cypress.io/api/events/catalog-of-events) event. Fixes [#25663](https://github.com/cypress-io/cypress/issues/25663).
 
+
```

Comment:
?

```suggestion
```

Reply:
Remnant of a merge conflict in the file where I had to move chunks around.